js/src/dom/manipulator.js

```diff
@@ -47,9 +47,13 @@ const Manipulator = {
       ...element.dataset
     }
 
-    Object.keys(attributes).forEach(key => {
-      attributes[key] = normalizeData(attributes[key])
-    })
+    Object.keys(attributes)
+      .filter(attributes => attributes.includes('bs'))
+      .forEach(key => {
+        let pureKey = key.replace('bs', '')
```

Comment:
Same here, if we know `key` starts with `'bs'` it might be better to do `replace(/^bs/, '')` so that reduce the chances of modifying something that isn't ours.

Reply:
Done :+1: